lib/sufia/import/generic_file_translator.rb

```diff
@@ -1,32 +1,20 @@
 module Sufia::Import
   # Imports a Sufia 6.0-exported GenericFile into a Sufia PCDM GernericWork and FileSet
-  class GenericFileTranslator
-    # @param Hash settings
+  class GenericFileTranslator < Translator
+    # @param Hash settings (see super for more)
     #   @attr import_binary    - (default true) Import the binary content from sufia6 fedora instance
     #     If true, sufia6_user and sufia6_password must be set in the appropriate sections of fedora.yml
     def initialize(settings)
+      super
+      @filename_prefix = settings.fetch(:filename_prefix, "generic_file_")
```

Comment:
@hackmastera I'm wondering if creating a method or an attribute that each class can override might make more sense than copying the fetch into each class.

In Translator
```
def default_prefix
  raise "Must override"
end
```

In this class
```
def default_prefix
  "generic_file_"
end
```

In Translator constructor:
```
@filename_prefix = settings.fetch(:filename_prefix, default_prefix)
```